ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -243,120 +243,102 @@
   </Form::Field::ListWrapper>
 
   {{! Account Claim Maps }}
-  <Rose::Form::Fieldset>
-    <:title>{{t 'form.account_claim_maps.label'}}</:title>
-    <:description>{{t 'form.account_claim_maps.help'}}</:description>
-    <:body>
-      {{#if @model.errors.account_claim_maps}}
-        <Rose::Form::Errors as |errors|>
-          {{#each @model.errors.account_claim_maps as |error|}}
-            <errors.message>{{error.message}}</errors.message>
-          {{/each}}
-        </Rose::Form::Errors>
-      {{/if}}
-      <Rose::Table @style='condensed' as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.from_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.to_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
+  <Hds::Form::Fieldset @isOptional={{true}} as |F|>
+    <F.Legend>{{t 'form.account_claim_maps.label'}}</F.Legend>
+    <F.HelperText>{{t 'form.account_claim_maps.help'}}</F.HelperText>
+    <F.Control>
+      <Hds::Table class='list-wrapper-field'>
+        <:head as |H|>
+          <H.Tr>
+            <H.Th id='fromClaimHeader'>{{t 'form.from_claim.label'}}</H.Th>
+            <H.Th id='toClaimHeader'>{{t 'form.to_claim.label'}}</H.Th>
+            <H.Th>{{t 'titles.actions'}}</H.Th>
+          </H.Tr>
+        </:head>
+        <:body as |B|>
           {{#each @model.account_claim_maps as |claimMap i|}}
-            <body.row as |row|>
-              <row.cell>
-                <form.input
-                  @type='text'
-                  @name='from_claim'
+            <B.Tr>
+              <B.Td>
+                <Hds::Form::TextInput::Field
                   @value={{claimMap.from}}
-                  @error={{@model.errors.account_claim_maps}}
-                  @contextual={{true}}
-                  as |field|
-                >
-                  <field.label>{{t 'form.from_claim.label'}}</field.label>
-                  <field.field />
-                </form.input>
-              </row.cell>
-              <row.cell>
-
+                  @type='text'
+                  disabled={{form.disabled}}
+                  aria-labelledby='fromClaimHeader'
+                  {{on 'input' (set-from-event claimMap 'from')}}
+                />
+              </B.Td>
+              <B.Td>
                 <Hds::Form::Select::Field
```

Comment:
Don't we need the `@isInvalid` here? wondering why we have it earlier and now we do not.

Reply:
@calcaide I am adding back the `@isInvalid` argument.  I have been trying to find documentation or patterns on errors on creation and updating.  I have been looking at the [auth_method_service.proto](https://github.com/hashicorp/boundary/blob/main/internal/proto/controller/api/services/v1/auth_method_service.proto), that gives you some detail on what messages are returned on those methods, but it seems we have `@isInvalid` on thing that may not even return an error message?

Reply:
@dvonanderson 
- Did you test editing/creating an OIDC Auth Method and adding Account Claim Maps with empty From Claim field?
- I have no knowledge of how the BE manages errors and check our requests when submitting a form, but curious why looking at that proto file, within there is no mention to the fields this form has.